nix/not-os-image.nix

```diff
@@ -0,0 +1,63 @@
+{ pkgs ? (import (import ../nix/sources.nix).nixpkgs { })
+, not-os ? (import ../nix/sources.nix).not-os
+}:
+let
+  config = (import not-os {
+    nixpkgs = pkgs.path;
+    configuration = { pkgs, ... }: {
+      imports = [
+        (not-os  + "/qemu.nix")
+      ];
+      environment.systemPackages = [
+        pkgs.utillinux
+        pkgs.gnugrep
+        pkgs.kmod
+      ];
+
+      not-os.nix = true;
```

Comment:
This is a smaller variant of NixOS: https://github.com/cleverca22/not-os
Builds & boots faster.